docs/docs/91-migrations.md

```diff
@@ -12,6 +12,15 @@ Some versions need some changes to the server configuration or the pipeline conf
 - Deprecated alternative names for secrets, use `environment` with `from_secret`
 - Deprecated slice definition for env vars
 - Deprecated `environment` filter, use `when.evaluate`
+- Deprecated some forge specific configs in favor of `WOODPECKER_FORGE_...`:
+  - Use `WOODPECKER_FORGE_URL` instead of	`WOODPECKER_GITHUB_URL`, `WOODPECKER_GITLAB_URL`, `WOODPECKER_GITEA_URL`, `WOODPECKER_BITBUCKET_URL`
+  - Use `WOODPECKER_FORGE_CLIENT` instead of `WOODPECKER_GITHUB_CLIENT`, `WOODPECKER_GITLAB_CLIENT`, `WOODPECKER_GITEA_CLIENT`, `WOODPECKER_BITBUCKET_CLIENT`, `WOODPECKER_BITBUCKET_DC_CLIENT_ID`
+  - Use `WOODPECKER_FORGE_SECRET` instead of `WOODPECKER_GITHUB_SECRET`, `WOODPECKER_GITLAB_SECRET`, `WOODPECKER_GITEA_SECRET`, `WOODPECKER_BITBUCKET_SECRET`, `WOODPECKER_BITBUCKET_DC_CLIENT_SECRET`
+  - Use `WOODPECKER_FORGE_SKIP_VERIFY` instead of `WOODPECKER_GITHUB_SKIP_VERIFY`, `WOODPECKER_GITLAB_SKIP_VERIFY`, `WOODPECKER_GITEA_SKIP_VERIFY`, `WOODPECKER_BITBUCKET_SKIP_VERIFY`
+- Use `WOODPECKER_DEV_OAUTH_HOST` instead of `WOODPECKER_DEV_GITEA_OAUTH_URL`
+- Replace `WOODPECKER_DEV_OAUTH_HOST` with `WOODPECKER_HOST` (public facing address) and `WOODPECKER_WEBHOOK_HOST` (internal webhook address)
```

Comment:
I don't understand, how webhook host is related to oAuth and public hosts?

Reply:
Before you've added #1869 we always told people to change `WOODPECKER_HOST` to the internal url and `WOODPECKER_DEV_OAUTH_HOST` to the public address. This was for example also done in the gitpod setup. Now after #1869 there is no reason anymore to have `WOODPECKER_HOST` different from `WOODPECKER_DEV_OAUTH_HOST` if you can also change the webhooks with `WOODPECKER_WEBHOOK_HOST`